src/index.js

```diff
@@ -52,6 +55,15 @@ export default class AutoCheckElement extends HTMLElement {
     this.setAttribute('csrf', value)
   }
 
+  get required(): boolean {
+    return this.hasAttribute('required')
```

Comment:
Should this be `this.hasAttribute('required') || this.input.hasAttribute('required')`? That way implementors could set the `required` on either the `<auto-check>` element or the `<input>`. It could prevent some frustrations when the `required` attribute is set on `<input>` and doesn't seem to be working?

Reply:
I think we should treat a required input value and required auto-check with the server as separate validations.